src/modal/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { View, TouchableOpacity, Text } from 'react-native';
+
+import { TextInput, Modal, Portal, Provider } from 'react-native-paper';
+import {
+  Container,
+  ModalContainer,
+  ModalInput,
+  TouchableButton,
+  TopView,
+  BottomView,
+  ModalTitle
+} from './styles';
+
+export default function ModalCard() {
+  return (
+    <Provider>
+      <Container>
+        <Portal>
+          <Modal
+            visible={true}
+            contentContainerStyle={{
+              alignItems: 'center',
+              backgroundColor: '#fff',
+              borderRadius: 10,
+              height: '50%',
+              width: '90%',
+              alignSelf: 'center'
+            }}
+          >
+            <TopView>
+              <ModalTitle>
+                Quickly setup a credit
+              </ModalTitle>
+              <ModalInput
+                theme={{ colors: { PRIMARY: '#000', placeholder: '#000' } }}
+                mode="outlined"
+                label="New Credit"
+                placeholder="Credit Amount"
+                placeholderTextColor="#00000040"
```

Comment:
change placeholder color to use app global colors, you can use the black color which is PRIMARY_TEXT and reduce its opacity to fit by converting it from hex color to RGB color using hexToRGB function located inside utils folder

```javascript
import React from 'react';
import { useAppTheme } from '../../theme';
import hexToRGB from '../../utils/hexToRGB';
import { NavigationInterface } from '../types';

// IMPORT FOR ALL CUSTOM STYLES
import { Container, Welcome } from './styles';

// DEFINE SCREEN PROP TYPES
interface ScreenProp extends NavigationInterface {}

export default function ProfileScreen(props: ScreenProp) {
  const { colors } = useAppTheme();

  return (
    <Container style={{ backgroundColor: hexToRGB(colors.PRIMARY, 0.5) }}>
      <Welcome>Home Screen Under Development</Welcome>
    </Container>
  );
}
```